.changeset/large-geese-play.md

```diff
@@ -0,0 +1,5 @@
+---
+"astro": minor
+---
+
+Exposes the `inferRemoteSize` function used by `inferSize`
```

Comment:
Can we have a description here that describes how this affects user behaviour? What it now allows them to do, or why this change was made that will help someone in their project?

With new features introduced in a minor we often do have slightly longer changelog messages, so even an example of using this would be helpful here!

Reply:
Done! 👍